contributing.md

```diff
@@ -0,0 +1,141 @@
+# Contributing
+
+> You want to contribute to the website? Welcome!
+> You will find here resources and advices that could help you.
+
+If you have any questions, you can ask them in #website-dev channel in our Discord.
+
+ * [Preparing the development environment](#preparing-the-development-environment)
+ * [Guidelines](#guidelines-1)
+ * [Localization](#localization)
+
+<!-- /code_chunk_output -->
+
+
+## Preparing the development environment
+### Fork the project
+* Fork the repo.
+* **Always create a new branch before working on your fork.**
+* You can now improve the website.
+
+> You can use a git client like [GitHub Desktop](https://desktop.github.com/) to manage your local repository.
+
+When your done submit a pull request.
+
+### Running the website
+> When you make modifications to the site it can be useful to render the code. You will thus see how the modifications look like.
+
+See [Building local enviroment](https://github.com/Paperback-iOS/website#building-local-enviroment)
+
+## Guidelines
+### Glossary
+
+ * App/application --> app
+ * Device/iPhone/iPad/phone --> iPhone or iPad
+ * iOS --> iOS or iPadOS
+
+> Be careful of capitalization:
+
+* Paperback
+* MangaDex
+
+* AltStore
+* AltServer
+
+* Safari
+* macOS
+
+### Syntax and layout
+> To create a numbered list use `1.` on all lines:
+```markdown
+1. Line 1
+1. Line 2
+```
+
+> Write `.IPA` in a code block.
+
+> Use `[Image]()` or `[GIF]()` to link to an explicative picture.
```

Comment:
Give an example of such

```suggestion
> You may render images and gifs by enclosing it in the following format: `[http://yourimagelink.jpg]()` or `[http://yourgiflink.gif]()`
```

Reply:
That wasn't really what I meant. The idea was: you can link to a picture by using a  `[Image](http://yourimagelink.jpg)` or `[GIF](http://yourgiflink.gif)` link.
The objective was to use a consistent form of link for pictures: always [Image](http://yourimagelink.jpg) or [GIF](http://yourgiflink.gif).

I think we could remove this information with #30. The idea is to add a Vue component that create a dialog windows.

Reply:
I like your additional component described in #32. I think I understand your intentions better now.
It's your call. I'd redact my suggestion, and suggest just sticking with what you've got initially if you don't decide to remove it.

Reply:
The component was added so this section is not needed anymore.